charts/jindocache/templates/fuse/daemonset.yaml

```diff
@@ -0,0 +1,247 @@
+{{- $notEnableDnsConfig := (eq .Values.master.dnsServer "1.1.1.1") }}
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "jindofs.fullname" . }}-fuse
```

Comment:
If the chart name is changed to jindocache, I think the name jindofs should be changed to jindocache.

Reply:
done